web-server/app/assets/javascripts/components/packages/show-package.jsx

```diff
@@ -29,6 +29,19 @@ define(function(require) {
     render: function() {
       var params = this.context.router.getCurrentParams();
       var rows = _.map(this.props.Package.deref(), function(value, key) {
+        if(key === "id") {
+          var idString = value.name + '-' + value.version;
+          return (
+            <tr key={idString}>
```

Comment:
I'd need to copy the prop, since it's read-only. Copying JS objects is not trivial: http://stackoverflow.com/a/728694/158703 . It will end up being more lines to write a copying routine. I could add a comment explaining why I'm not just reassigning?